### DISCUSSION TITLE
GET request for filter controller

### DISCUSSION THREAD

**Discussion by srapilly** [sidekiq/sidekiq#6402] [Ideas]
Hi,

Currently all of the filter forms are using the `method="post"`. In my company,  `GET` requests are allowed for everyone, and `POST` only for certain users (we could also just update our policy to allow certain paths)

It could be nice if sidekiq use a `GET` request for this, it will also be easier to share a URL from a filter controller.

---

The controller is already working with a GET request:

<img width="1166" alt="image" src="https://github.com/user-attachments/assets/fe94356e-e0be-4c36-84ed-a0808f18a86a">

**Comment by mperham** [comment]
GETs are meant to be idempotent and cached but search results can change second by second and shouldn’t be cached. The GET support is there for people to use in e.g. a bookmark, but the form implementation uses POST for those reasons.
